gtda/externals/python/cubical_complex_interface.py

```diff
@@ -1,9 +1,9 @@
 import os
 import numpy as np
-from giotto_cubical_complex \
+from gtda.externals.modules.giotto_cubical_complex \
```

Comment:
I'll stop saying it now, please change everywhere else if it's a good idea.

Reply:
Well after giving a better look on which will be best practice inside a library, this [link](https://stackoverflow.com/a/16748366) summarize quite well.

Relative import is used because of, from [PEP 328](https://www.python.org/dev/peps/pep-0328/#rationale-for-relative-imports)

> Several use cases were presented, the most important of which is being able to rearrange the structure of large packages without having to edit sub-packages. In addition, a module inside a package can't easily import itself without relative imports.

A caveat of using this method is that you cannot execute a `file.py` that has a relative import or you'll get a `ValueError: Attempted relative import in non-package`. To overcome this issue, you'll require to execute it as a module:

```bash
python3 -m gtda.externals.python.cubical_complex_interface
```

I think that we can move to relative imports, but need to understand that it adds a limitation.

Reply:
Thanks for the research! I hadn't thought so deeply about it before. My comment was mostly due to the fact that we use relative imports everywhere else, with the only exceptions of the test folders.

Reply:
If it's used everywhere else in the library, then we should keep consistency.
Can I proceed to the changes ?

Reply:
Go ahead! :)